.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ aliases:
     executor:
       name: android/android-docker
       resource-class: large
-      tag: 2023.02.1
+      tag: 2023.06.1
```

Comment:
I noticed that if SDKMAN failed to install the java sdk, it would fail to build since with the new gradle plugin, java 17 is required. I updated to the latest android executor to have java 17 installed out of the box.